Base/QTApp/qSlicerMainWindow.cxx

```diff
@@ -605,30 +605,34 @@ QList<qSlicerIO::IOProperties> qSlicerMainWindowPrivate::readRecentlyLoadedFiles
 {
   QList<qSlicerIO::IOProperties> fileProperties;
 
-  QSettings settings;
-  int size = settings.beginReadArray("RecentlyLoadedFiles/RecentFiles");
+  ctkCoreSettings* settings = qSlicerCoreApplication::application()->userSettings();
```

Comment:
I think  we can  keep it as `ctkCoreSettings settings;`

Reply:
We would need to copy the application home directory from usersettings object and I don't think it is copied (I think copying is disabled for ctkCoreSettings due to the private member implementation class).

Reply:
To address this, may be we could also set the home directory as a dynamic property to the application instance?